### PR TITLE
Update Bold and other smaller fixes

### LINKED ISSUE #268: Feature Request: Grave and Acute tones marks accessibility improvement
# Description of the new feature/enhancement (with images if possible)

First, thank you very much for fixing that issue with incorrect combining diacritic marks placement. Everything works correctly now, but I noticed that combining Grave (#0340) and Acute (#0341) tones' marks are hardly distinguished from a dot over a standard small "i" when font size is not large (<=12). It would be great to modify these marks by increasing their slopes and making upper parts of these symbols a bit thicker like in this picture.
accent diactrics
There are two variants - upper one using Cascadia Code (10pt) and the lower one using Calibri (11pt), image is scaled to make it easier to notice the difference.

# Proposed technical implementation details (optional)

Increase the slope of these tone marks and make them thicker in the upper parts (see an example with accented small "i" in the end of the first row - the first small "i" with Grave is a current glyph, the second glyph is a proposed option.)

### LINKED ISSUE #282: Rendering for Unicode Caret
# Environment

```none
wt 0.11.1333
OS platform and version: Windows 10 1903
```

# Steps to reproduce

Paste this string onto a wsl bash prompt, or a Julia REPL (win/linux) inside terminal: `α̂=1`

The four characters in this string are
```
U+03B1 : GREEK SMALL LETTER ALPHA
U+0302 : COMBINING CIRCUMFLEX ACCENT {hat}
U+003D : EQUALS SIGN
U+0031 : DIGIT ONE
```

# Expected behavior

With Fira Code, size 12

image

# Actual behavior

With Cascadia Mono, 12

image

With Cascadia, 12
image

_Note_: If testing on powershell prompt, the display is incorrect even with Fira Code, but look at the rendering of the PS error message, where the display is correct with Fira, but incorrect with Cascadia

### LINKED ISSUE #294: Prime and Double Prime should be better distinguishable from Apostrophe and Quotation mark
# Environment

```none
Cascadia Code version number: 2004.300
Application (with version) used to display text: Settings App
OS platform and version: Windows 1909
Screen resolution (i.e. 220dpi): 100%

Any other software? VS 2019
```

# Steps to reproduce

image

I'm using the Double Prime character `U+2033` to denote "inches" in texts and that's part of a string in C# source code, so it's surrounded by quotation marks `U+0022`. I noticed that these two characters, especially when put next to each other, are nearly indistinguishable:

image

### PR DESCRIPTION
**Pull request by aaronbell**
This PR shifts the position of the "Bold" instance closer to the Regular
in weight.

Additionally, I've fixed a couple of other smaller issues around
diacritic marks and symbols.

In conducting research into Visual Studio's rendering of Cascadia Code
(versus other fonts), I began to feel that the Bold that was in use felt
a bit too heavy. In selecting a lighter point on the axis, I was able to
achieve better rendering results in VS and without drawing as much
attention to the Bold. 

The main issue with VS is that it does not support y-direction
smoothing, so overly bold Glyphs start to look quite distorted and
weird. Additionally, it offers only minimal variable font support so I
can’t STAT my way out of it ;).

Other fixes:
* Added linked bolds for the lighter weights.
* Tweaked design of primes to give a more angled feel.
* Added weight to the top of the acutecomb and gravecomb glyphs
* Added anchors to the common lowercase Greek characters. 
* Corrected build script to correctly build TTF variable fonts. 
* Resolved some introduced hinting bugs

VALIDATION

Reviewed in VS, VS Code, notepad. Validated no impact on hinting
instructions. 

Closes #294 - changed design of primes to make them appear more angled.
Closes #282 - added anchors to Greek for use in Julia REPL
Closes #268 - added additional weighting to grave / acute to help
differentiate from dot

### CODE REVIEW

**Comment by aaronbell** [comment]
Hm. Reconsidering the Bold set up. Feel like Semi and Bold are too similar...

**Comment by aaronbell** [comment]
Ready for you @DHowett

**Comment by DHowett** [comment]
Great! So, this lightens Bold a bit overall? I'm pretty comfortable with that :smile:

**Comment by aaronbell** [comment]
Yup! Just a little bit lighter. It will help a little with the crummy situation in VS.

File: build.py [hunk 1]
```diff
@@ -151,23 +151,23 @@ def build_variable_fonts(designspace, *steps):
 
     print(f"[{familyName}] Done: {file_path}")
 
-    print(f"[{familyName}] Compiling CFF2")
-    file_path_cff2 = (OUTPUT_DIR / file_stem).with_suffix(f".otf")
+    #print(f"[{familyName}] Compiling CFF2")
```
Comment (DHowett) [inline build.py:154]: oop

Comment (aaronbell) [reply build.py]: :waves hand: These aren't the variable fonts you're looking for.

**Comment by aaronbell** [comment]
Reverted PL modification

**Comment by DHowett** [comment]
As always, thanks!

**Pull request merged**